adios2 error during build

Happens with `adios2` v2.9.0:

```
    compiling: adiosCoreWrite.C
write/adiosCoreWrite.C:77:17: error: no matching constructor for initialization of 'adios2::ADIOS'
            new adios2::ADIOS
                ^
/Volumes/OpenFOAM-v2212/usr/Cellar/adios2/2.9.0/include/adios2/cxx11/ADIOS.h:67:5: note: candidate constructor not viable: cannot convert argument of incomplete type 'MPI_Comm' (aka 'ompi_communicator_t *') to 'const std::string' (aka 'const basic_string<char>') for 1st argument
    ADIOS(const std::string &configFile, MPI_Comm comm);
    ^
/Volumes/OpenFOAM-v2212/usr/Cellar/adios2/2.9.0/include/adios2/cxx11/ADIOS.h:106:5: note: candidate constructor not viable: cannot convert argument of incomplete type 'MPI_Comm' (aka 'ompi_communicator_t *') to 'const std::string' (aka 'const basic_string<char>') for 1st argument
    ADIOS(const std::string &configFile, const std::string &hostLanguage);
    ^
/Volumes/OpenFOAM-v2212/usr/Cellar/adios2/2.9.0/include/adios2/cxx11/ADIOS.h:56:5: note: candidate constructor not viable: requires single argument 'comm', but 2 arguments were provided
    ADIOS(MPI_Comm comm);
    ^
/Volumes/OpenFOAM-v2212/usr/Cellar/adios2/2.9.0/include/adios2/cxx11/ADIOS.h:89:5: note: candidate constructor not viable: requires single argument 'configFile', but 2 arguments were provided
    ADIOS(const std::string &configFile);
    ^
/Volumes/OpenFOAM-v2212/usr/Cellar/adios2/2.9.0/include/adios2/cxx11/ADIOS.h:114:5: note: candidate constructor not viable: requires 1 argument, but 2 were provided
    ADIOS(const ADIOS &) = delete;
    ^
/Volumes/OpenFOAM-v2212/usr/Cellar/adios2/2.9.0/include/adios2/cxx11/ADIOS.h:120:5: note: candidate constructor not viable: requires 1 argument, but 2 were provided
    ADIOS(ADIOS &&) = default;
    ^
/Volumes/OpenFOAM-v2212/usr/Cellar/adios2/2.9.0/include/adios2/cxx11/ADIOS.h:78:5: note: candidate constructor not viable: requires 3 arguments, but 2 were provided
    ADIOS(const std::string &configFile, MPI_Comm comm,
    ^
/Volumes/OpenFOAM-v2212/usr/Cellar/adios2/2.9.0/include/adios2/cxx11/ADIOS.h:95:5: note: candidate constructor not viable: requires 0 arguments, but 2 were provided
    ADIOS();
    ^
write/adiosCoreWrite.C:88:17: error: no matching constructor for initialization of 'adios2::ADIOS'
            new adios2::ADIOS
                ^
/Volumes/OpenFOAM-v2212/usr/Cellar/adios2/2.9.0/include/adios2/cxx11/ADIOS.h:67:5: note: candidate constructor not viable: cannot convert argument of incomplete type 'MPI_Comm' (aka 'ompi_communicator_t *') to 'const std::string' (aka 'const basic_string<char>') for 1st argument
    ADIOS(const std::string &configFile, MPI_Comm comm);
    ^
/Volumes/OpenFOAM-v2212/usr/Cellar/adios2/2.9.0/include/adios2/cxx11/ADIOS.h:106:5: note: candidate constructor not viable: cannot convert argument of incomplete type 'MPI_Comm' (aka 'ompi_communicator_t *') to 'const std::string' (aka 'const basic_string<char>') for 1st argument
    ADIOS(const std::string &configFile, const std::string &hostLanguage);
    ^
/Volumes/OpenFOAM-v2212/usr/Cellar/adios2/2.9.0/include/adios2/cxx11/ADIOS.h:56:5: note: candidate constructor not viable: requires single argument 'comm', but 2 arguments were provided
    ADIOS(MPI_Comm comm);
    ^
/Volumes/OpenFOAM-v2212/usr/Cellar/adios2/2.9.0/include/adios2/cxx11/ADIOS.h:89:5: note: candidate constructor not viable: requires single argument 'configFile', but 2 arguments were provided
    ADIOS(const std::string &configFile);
    ^
/Volumes/OpenFOAM-v2212/usr/Cellar/adios2/2.9.0/include/adios2/cxx11/ADIOS.h:114:5: note: candidate constructor not viable: requires 1 argument, but 2 were provided
    ADIOS(const ADIOS &) = delete;
    ^
/Volumes/OpenFOAM-v2212/usr/Cellar/adios2/2.9.0/include/adios2/cxx11/ADIOS.h:120:5: note: candidate constructor not viable: requires 1 argument, but 2 were provided
    ADIOS(ADIOS &&) = default;
    ^
/Volumes/OpenFOAM-v2212/usr/Cellar/adios2/2.9.0/include/adios2/cxx11/ADIOS.h:78:5: note: candidate constructor not viable: requires 3 arguments, but 2 were provided
    ADIOS(const std::string &configFile, MPI_Comm comm,
    ^
/Volumes/OpenFOAM-v2212/usr/Cellar/adios2/2.9.0/include/adios2/cxx11/ADIOS.h:95:5: note: candidate constructor not viable: requires 0 arguments, but 2 were provided
    ADIOS();
    ^
2 errors generated.
make[1]: *** [/Volumes/OpenFOAM-v2212/build/darwin64ClangDPInt32Opt/modules/adios/src/adiosFoam/write/adiosCoreWrite.o] Error 1
```

--- a/Brewfile
+++ b/Brewfile
@@ -4,14 +4,16 @@ tap "gerlero/openfoam"
 brew "open-mpi"
 brew "libomp"
 
-# Optional dependencies
-brew "adios2"
+# Optional dependencies (enabled by default)
 brew "boost"
 brew "fftw"
 brew "kahip"
 brew "metis"
 brew "gerlero/openfoam/cgal@4"
 brew "gerlero/openfoam/scotch-no-pthread"
+
+# Optional dependencies (uncomment to enable)
+# brew "adios2"
 
 # Optional tools
 brew "bash"

